js/index.js

```diff
@@ -0,0 +1,136 @@
+function showhide() {
+  const element = document.querySelector('.button');
+  const closemenu = document.querySelector('.mobile-menu');
+  if (element.style.display !== 'none') {
+    element.style.display = 'none';
+    closemenu.style.visibility = 'visible';
+    closemenu.style.height = '60%';
+  } else if (element.style.display === 'none') {
+    closemenu.style.height = '0';
+    setTimeout(() => {
+      closemenu.style.visibility = 'hidden';
+      element.style.display = 'block';
+    }, 200);
+  }
+}
+
+const openMenu = document.querySelector('#logo');
+openMenu.addEventListener('click', showhide);
+
+const closeMenu = document.querySelector('.menu-button');
+closeMenu.addEventListener('click', showhide);
+
+const array = [{
+  name: 'Joanna Popper (LA)',
```

Comment:
- [ ] You did well 😃 at adding the speakers in the section, but I see the size of the names of the speakers are bigger than the Behance Design, so please try to decrease the font size to make it like the Behance 😊


> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225888014-e75c3e0e-f165-4ace-8c67-de5b17e63d0a.png) |   ![image](https://user-images.githubusercontent.com/83360636/225887933-358d8671-7cbd-4e3f-a55a-c74ad40a2817.png)   |